msgcntr/messageforums-app/src/webapp/jsp/dfReviseForumSettingsAttach.jsp

```diff
@@ -295,19 +294,19 @@
                    <h:inputText id="closeDate" styleClass="closeDate" value="#{ForumTool.selectedForum.closeDate}" onchange="storeCloseDateISO(event)"/>
                    <h:inputText id="closeDateISO" styleClass="closeDateISO hidden" value="#{ForumTool.selectedForum.closeDateISO}"></h:inputText>
                </h:panelGroup>
-				<p class="checkbox">
+				<h:panelGroup layout="block" styleClass="checkbox" style="display: #{ForumTool.doesSiteHaveCalendar ? '' : 'none'}">
```

Comment:
Please test it, I am not sure if jsf styleClass supports conditions
```suggestion
				<h:panelGroup layout="block" styleClass="checkbox #{ForumTool.doesSiteHaveCalendar ? '' : 'd-none'}">
```

Reply:
no, this doesn't work